Dockerfile

```diff
@@ -84,6 +85,7 @@ RUN ./configure \
 	--enable-subnet \
 	--with-pthreads \
 	--with-libevent \
+	--with-libhiredis \
```

Comment:
Do we also need --enable-cachedb here @hugochc ? See #408

Reply:
Hi @klutchell , sorry for late response. Yes, cachedb option should be enable in Dockerfile.